lib/libutee/include/tee_api_defines.h

```diff
@@ -79,6 +79,7 @@
 #define TEE_PARAM_TYPE_VALUE_INPUT      1
 #define TEE_PARAM_TYPE_VALUE_OUTPUT     2
 #define TEE_PARAM_TYPE_VALUE_INOUT      3
+#define TEE_PARAM_TYPE_MEMREF_SECURE    4
```

Comment:
Since it's our own extension, this should be in `lib/libutee/include/tee_api_defines_extensions.h`. And I'm not sure about the value, `4` is almost certainly wrong, it looks like it was used by GP then deprecated? (I could not find a reference supporting my claim, though).